src/main/java/uk/gov/hmcts/reform/sscs/service/SendNotificationService.java

```diff
@@ -215,7 +215,7 @@ private static boolean isValidLetterAddress(Address addressToUse) {
     }
 
     private void sendBundledLetterNotificationToAppellant(NotificationWrapper wrapper, Notification notification, SubscriptionType subscriptionType) {
-        if (hasLetterTemplate(notification)) {
+        if (hasBundledLetterTemplate(wrapper)) {
```

Comment:
We don't need this check, as bundled letters don't have templates.  They exist in sscs-pdf-email-common